tools/prune-old-builds.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+
+# Copyright (C) 2020  Matthew "strager" Glazar
+# See end of file for extended copyright information.
+
+import os
+import sys
+import shutil
+import argparse
+import subprocess
+from pathlib import Path
+from datetime import datetime
+
+
+FOURTEEN_DAYS = 14
+
+
+def how_old(date: datetime) -> int:
+    """Return a integer that represents how many days has passed given a date"""
+    return (datetime.now() - date).days
+
+
+def error_print(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+
+def get_commits(repo_name: str, repository_url: str) -> list:
+    try:
+        return subprocess.check_output(
+            f"cd {repo_name} \
+            && git fetch --prune {repository_url} '+refs/pull/*/head:refs/remotes/github-pr/*' \
```

Comment:
If `repository_url` contains characters interpreted by the shell, this will do funky things.

I recommend avoiding `shell=True`. If we can't avoid it, shell-escape the string:
```suggestion
            && git fetch --prune {pipes.quote(repository_url)} '+refs/pull/*/head:refs/remotes/github-pr/*' \
```